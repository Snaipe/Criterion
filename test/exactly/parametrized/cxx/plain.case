[act]

% parameterized.cc.bin

[assert]

stdout is-empty

stderr equals -contents-of expected/plain.txt
