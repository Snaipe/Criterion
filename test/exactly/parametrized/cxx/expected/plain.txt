[----] parameterized.cc:72: Assertion Failed
[----]   
[----]   Parameters: (1, 2.000000)
[----]   
[FAIL] params::cleanup
[----] parameterized.cc:72: Assertion Failed
[----]   
[----]   Parameters: (3, 4.000000)
[----]   
[FAIL] params::cleanup
[----] parameterized.cc:72: Assertion Failed
[----]   
[----]   Parameters: (5, 6.000000)
[----]   
[FAIL] params::cleanup
[----] parameterized.cc:34: Assertion Failed
[----]   
[----]   Parameters: (1, 2.000000)
[----]   
[FAIL] params::multiple
[----] parameterized.cc:34: Assertion Failed
[----]   
[----]   Parameters: (3, 4.000000)
[----]   
[FAIL] params::multiple
[----] parameterized.cc:34: Assertion Failed
[----]   
[----]   Parameters: (5, 6.000000)
[----]   
[FAIL] params::multiple
[----] parameterized.cc:13: Assertion Failed
[----]   
[----]   Parameter: 1
[----]   
[FAIL] params::simple
[----] parameterized.cc:13: Assertion Failed
[----]   
[----]   Parameter: 2
[----]   
[FAIL] params::simple
[----] parameterized.cc:13: Assertion Failed
[----]   
[----]   Parameter: 3
[----]   
[FAIL] params::simple
[----] parameterized.cc:92: Assertion Failed
[----]   
[----]   string: Hello
[----]   
[FAIL] params::string
[----] parameterized.cc:92: Assertion Failed
[----]   
[----]   string: World
[----]   
[FAIL] params::string
[====] Synthesis: Tested: 11 | Passing: 0 | Failing: 11 | Crashing: 0 
